### PR TITLE
Linked

### PR DESCRIPTION
**Pull request by eden-fenster**
Managed to link between the web and the query.

### CODE REVIEW

File: web/templates/upload.html [hunk 1]
```diff
@@ -16,7 +16,7 @@ <h4>1. Upload a text file that has an unsolved sudoku in it here.</h4>
       </form>
       <h4>2. Wait.</h4>
       <h4>3. Enjoy your solved sudoku.</h4>
-   <form id = "database" action="http://sudoku_query:1000/">
+   <form id = "database" action="http://10.0.2.15:1000/">
```
Comment (fenster-software) [inline web/templates/upload.html]: Why is using a hard coded IP address better?

Comment (eden-fenster) [reply web/templates/upload.html]: Because if I don't then it won't connect

Comment (fenster-software) [reply web/templates/upload.html]: @eden-fenster Please read up on DNS and Docker.

Comment (fenster-software) [reply web/templates/upload.html]: Using DNS means that you use names in your code, not numbers. I think that we need a directed discussion about this.

**Comment by eden-fenster** [comment]
Please review the changes and the comments in the commit message

File: web/templates/upload.html [hunk 2]
```diff
@@ -16,9 +16,11 @@ <h4>1. Upload a text file that has an unsolved sudoku in it here.</h4>
       </form>
       <h4>2. Wait.</h4>
       <h4>3. Enjoy your solved sudoku.</h4>
-   <form id = "database" action="http://sudoku_query:1000/">
-         <button type="submit">Search in database</button>
-      </form>
+      <a id = "database" href="http://sudoku_query:1000/" target="_self">
```
Comment (fenster-software) [inline web/templates/upload.html]: What is the "a" HTML?

**Pull request merged**